src/wolff_producer.erl

```diff
@@ -456,11 +466,14 @@ mark_connection_down(#{topic := Topic,
   case is_idle(St) of
     true ->
       %% nothing to send, will trigger reconnect later
+      log_info(Topic, Partition, "lost connection to partition leader while idling, "
```

Comment:
Will this trigger lots of logging? Could we use counters instead?

Reply:
This is to make the log symmetric.
There is a log for retry, but not one down.
When it comes to number of logs:
It’s not going to be more than the other exiting reconnect attempt log.